Source/WebKit/UIProcess/GPU/GPUProcessProxy.cpp

```diff
@@ -270,7 +270,9 @@ void GPUProcessProxy::setOrientationForMediaCapture(WebCore::IntDegrees orientat
 #if HAVE(APPLE_CAMERA_USER_CLIENT)
 static const ASCIILiteral appleCameraUserClientPath { "com.apple.aneuserd"_s };
 static const ASCIILiteral appleCameraUserClientIOKitClientClass { "H11ANEInDirectPathClient"_s };
+// FIXME: Remove the "H11ANEIn" extension when both are no longer needed.
```

Comment:
We might need a radar for removal, with description about when this will no longer be needed (e.g. does it depend on a service we will remove?), as a reminder for future cleanup.

Reply:
That's a good point -- I added a radar in the latest version.

Thanks for reviewing!